be/src/olap/tablet.cpp

```diff
@@ -839,6 +839,12 @@ bool Tablet::check_path(const std::string& path_to_check) {
             return true;
         }
     }
+    for (auto& inc_version_rowset : _inc_rs_version_map) {
+        bool ret = inc_version_rowset.second->check_path(path_to_check);
+        if (ret) {
```

Comment:
maybe only check rowset id not the full path?

Reply:
OK， I will add check_rowset_id to do this